SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp

```diff
@@ -77,26 +95,27 @@ struct ConfigureConnection
 		Params.enable_dynamic_components = true;
 	}
 
-	FString FormatProtocolPrefix() const
+	FString FormatLogFilePrefix() const
 	{
-		FString FinalProtocolLoggingPrefix = FPaths::ConvertRelativePathToFull(FPaths::ProjectLogDir());
-		if (!Config.ProtocolLoggingPrefix.IsEmpty())
+		FString FinalLogFilePrefix = FPaths::ConvertRelativePathToFull(FPaths::ProjectLogDir());
```

Comment:
Isn't this being used for the `WorkerLogPrefix`, is there now no difference between this and protocol logging?

Reply:
As written above, yes, both go into the same file but you can choose what to log with the command line parameters.